helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/IdleMessage.java

```diff
@@ -0,0 +1,53 @@
+package org.symphonyoss.symphony.bots.helpdesk.messageproxy;
+
+import org.apache.commons.codec.binary.Base64;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.symphony.bots.helpdesk.messageproxy.config.HelpDeskBotInfo;
+import org.symphonyoss.symphony.bots.helpdesk.messageproxy.config.HelpDeskServiceInfo;
+import org.symphonyoss.symphony.bots.helpdesk.messageproxy.config.IdleTicketConfig;
+import org.symphonyoss.symphony.bots.helpdesk.messageproxy.message.IdleMessageBuilder;
+import org.symphonyoss.symphony.bots.helpdesk.messageproxy.service.TicketService;
+import org.symphonyoss.symphony.bots.helpdesk.service.model.Ticket;
+import org.symphonyoss.symphony.clients.model.SymMessage;
+
+@Component
+public class IdleMessage {
```

Comment:
Please rename this class to 'IdleMessageService' or something like that. You should also include javadocs and unit tests.

Reply:
ok